package.json

```diff
@@ -20,7 +20,7 @@
     "check": "tslint --project . && tsc --noEmit --pretty",
     "clean": "polkadot-dev-clean-build",
     "postinstall": "polkadot-dev-yarn-only",
-    "test": "jest --coverage"
+    "test": "jest --coverage --config='jest.config.js'"
```

Comment:
Why the extra config here?

Reply:
i'll remove it.  i had added it so it would also run the tests that i added in the docs/examples subdirectories, but those tests have all been removed

Reply:
i'll remove it.  i had added it so it would also run the tests that i added in the docs/examples subdirectories, but those tests have all been removed